deploy/templates/notificationtemplates/sandbox/idlertriggered/notification.html

```diff
@@ -41,22 +41,21 @@
 >
 
     <p>
-        You are receiving this email because one or more of your workloads in the Developer Sandbox for
-        Red Hat OpenShift has been running for 12 hours.
+        You are receiving this email because one or more of your workloads in the Developer Sandbox has been running for 12 hours.
     </p>
 
     <p>
-        In accordance with the usage terms of Developer Sandbox, your workload {{.AppType}} {{.AppName}} has been scaled down. You can restart your workload(s) or scale up the number of instances from the
-        Developer Sandbox User Interface.
+        After 12 hours, our system idles running workloads. You can <a href="https://developers.redhat.com/learn/openshift/revive-inactive-openshift-pods-scaled-zero">restart your workload,</a> {{.AppType}} {{.AppName}}
+        when you're ready to continue exploring on the Developer Sandbox.
```

Comment:
This suggestion is tied to the suggestion above just to make it a bit more clear that the user doesn't need to do anything.
```suggestion
        After 12 hours, our system idles running workloads. No further action is required by you. You can <a href="https://developers.redhat.com/learn/openshift/revive-inactive-openshift-pods-scaled-zero">restart your workload,</a> {{.AppType}} {{.AppName}}
        when you're ready to continue exploring on the Developer Sandbox.
```

Reply:
I like this one too